### ISSUE TITLE
[Feature Request] Configurable date text format in commit detail and commit list

### ISSUE THREAD

**Issue by doneykoo** [ForkIssues/TrackerWin#1254]
I'd like to have Configurable date text format, e.g. like: `2021-07-28 20:39:13`
, instead of 
> 28 7月 2021 20:39:13 +08:00
> 28 Jul 2021 20:39

**Comment by carun** [comment]
Hiding the time-zone information in commit detail may not be a good idea. The default behavior seems perfect, to me.

**Comment by doneykoo** [comment]
> Hiding the time-zone information

Agreed. Of course the "time-zone" can be kept as it is where applicable in current version.

I mean the key point is not to hide "the time-zone information".
The request is to allow user config it to be `YYYY-MM-DD` numbers, just like ISO format, like `git log --date=iso-local` does. (Instead of always being RFC format.)

**Comment by starryspace** [comment]
Same issue.

**Comment by BenAtPws** [comment]
This would still be a nice to have feature.  Additionally, I would like to be able to have it display date/times in my time zone.  We have people across 3 time zones working on a shared project, and while having the time zone offset is great, local time would be helpful.
